assets/dev/scss/frontend/general/_button.scss

```diff
@@ -70,6 +69,8 @@
 		}
 	}
 
+	// TODO: Delete in Elementor 3.24 when all buttons migrated from `order` to `flex-direction`.
+	// For now, it's here for BC, for sites that upgrade Elementor, but not Elementor Pro.
```

Comment:
Initially, this entire style block (`.elementor-align-icon-*`) was deleted. But to be extra safe, we will keep it for a few releases.